parts/k8s/windowslogscleanup.ps1

```diff
@@ -0,0 +1,25 @@
+<#
+.DESCRIPTION
+    This script cleans old rotated logs for various kubernetes components.
+#>
+
+$global:LogPath = "c:\k\windowslogscleanup.log"
```

Comment:
that's captured in https://github.com/Azure/aks-engine/blob/3a2dcb2cb89fab9b05d50ff52a64e1d2155d82ac/scripts/collect-windows-logs.ps1#L7 - good